src/rev_ai/streamingclient.py

```diff
@@ -72,7 +72,8 @@ def start(self,
               detailed_partials=None,
               start_ts=None,
               transcriber=None,
-              language=None):
+              language=None,
+              skip_postprocessing=None):
```

Comment:
Why is it None rather than False?

Reply:
I did it mostly for consistency with the other boolean parameters. The reasoning I see for making it None is that it's not the SDKs place to assume what the API defaults are so it should default to all parameters being empty